src/database/sql_database.cc

```diff
@@ -1032,7 +1031,7 @@ std::shared_ptr<CdsObject> SQLDatabase::findObjectByPath(fs::path fullpath, bool
     else
         dbLocation = addLocationPrefix(LOC_DIR_PREFIX, fullpath);
 
-    std::vector<std::string> where {
+    auto where = std::array {
```

Comment:
Local testing shows this to increase size rather than reduce it...

I guess std::array mostly makes sense in constexpr context.